### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by diniiharry**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,5 +7,6 @@
     <title>Welcome to Potionfy</title>
 </head>
 <body>
+    <h1>Welcome to Potionfy! Enjoy your stay</h1>
```
Comment (diniihermoine) [inline index.html]: Could you remove the "Enjoy your stay" string and make the "Welcome to Potionfy!" string red?

**Review by diniihermoine — COMMENTED** [review]
done

**Comment by diniiharry** [comment]
sure, can do!

**Review by diniihermoine — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review

**Review by diniihermoine — APPROVED** [review]

**Pull request merged**